src/assets/scripts/client/navigationLibrary/FixCollection.js

```diff
@@ -162,8 +180,11 @@ class FixCollection extends BaseCollection {
      * @private
      */
     _buildFixModelsFromList(fixList, referencePosition) {
-        _forEach(fixList, (fixCoordinates, fixName) => {
-            const fixModel = new FixModel(fixName, fixCoordinates, referencePosition);
+        _forEach(fixList, (fixData) => {
+            if (!fixData.spoken) {
+                fixData.spoken = null; // the initializer for FixModel handles this correctly
```

Comment:
Maybe this is unnecessary. Is it already `null` or `undefined`? (comment for myself)

Reply:
Most likely `undefined` if it is not in the JSON. I have changed it and tested and it seems to work.